boardswarm/src/main.rs

```diff
@@ -423,13 +424,43 @@ trait Device: Send + Sync {
     fn current_mode(&self) -> Option<String>;
 }
 
+macro_rules! impl_u64_index {
+    ($id:ident) => {
+        #[derive(Copy, Clone, Debug, Default, Hash, PartialOrd, Ord, PartialEq, Eq)]
+        pub struct $id(u64);
+
+        impl Display for $id {
+            fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+                write!(f, "Actuator {}", self.0)
```

Comment:
The string here was left hardcoded to "Actuator"

Reply:
Nice catch, should be fixed now :)

Reply:
Ack, I think it's ready to be merged then.